src/plugins/intel_cpu/src/.clang-tidy

```diff
@@ -49,6 +52,10 @@ Checks: >
   -google-readability-casting,
   -google-readability-todo,
   -readability-braces-around-statements,
+  -modernize-avoid-c-arrays,
+  -modernize-loop-convert,
```

Comment:
I guess not necessary anymore.
Please double check the changes made in this config file

Reply:
Agree, revised the list of other `.clang-tidy` file changes as well.